server/server.js

```diff
@@ -72,8 +87,10 @@ module.exports = async ({graphQLConfig, graphiqlConfig, postgresConfig, schemaLi
 
       if (newSchema) {
         schema = newSchema.schema
-        subscriptionServer.close()
-        subscriptionServer = newSubscriptionServer(server, schema)
+        const newApp = newExpressApp(schema, server)
```

Comment:
I think integration tests are failing because the tracing and playground config aren't being passed here is all.

Reply:
Yep just spotted the same. Fixed now and testing again. Making some more changes will ping you shortly. Thanks.